$MirOS: ports/print/cups/patches/patch-filter_imagetops_c,v 1.2 2006/06/21 19:25:35 bsiegert Exp $
--- filter/imagetops.c.orig	Sat Jul 12 00:48:26 2008
+++ filter/imagetops.c	Tue Oct 21 22:56:23 2008
@@ -545,7 +545,7 @@ main(int  argc,				/* I - Number of comm
     * Set the new custom size...
     */
 
-    sprintf(s, "Custom.%.0fx%.0f", width, length);
+    snprintf(s, sizeof(s), "Custom.%.0fx%.0f", width, length);
     ppdMarkOption(ppd, "PageSize", s);
 
    /*
